test/cpp/rpc/test_wire_serialization.cpp

```diff
@@ -37,3 +37,16 @@ TEST(WireSerialize, Base) {
   run("hi", {torch::randn({5, 5})});
   run("more", {torch::randn({5, 5}), torch::rand({10, 10})});
 }
+
+TEST(WireSerialize, RecopySparseTensors) {
+  // Take a 1K row of a 1M tensors, and make sure we don't send across 1M rows.
+  static const size_t k1K = 1024;
```

Comment:
Nit: Prefer using constexpr instead of static const.

Reply:
Just replaced a few constexpr
Sorry, 'static const' is a bit of an ingrained habit, like C-x C-s/etc :)